docs_src/microservices/security/edgex-secrets-config-proxy.md

```diff
@@ -0,0 +1,142 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
+
+# NAME
+
+**edgex-secrets-config-proxy** – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-config** proxy [OPTIONS] SUBCOMMAND [ARG...]
+
+# DESCRIPTION
+
+**edgex-secrets-config-proxy** configures the EdgeX proxy service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and relevant utility functions.
+
+These commands require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **-m** _/path/to/resp-init.json_ (optional)
+
+    Points to location of secret store master key. Defaults to /vault/config/assets/resp-init.json.
+
+# COMMANDS
+
+  * **tls**
+
+    Configure inbound TLS certificate. This command will configure the TLS secrets into the secret store and re-deploy them to Kong. Requires additional arguments:
+
+    * --certificate _/path/to/certchain.pem_ (required)
+
+      Path to TLS leaf certificate (with optional appended certificate chain, not including a root certificate).
+
+    * --key _/path/to/private.pem_ (required)
+
+      Path to TLS private key.
+
+  * **adduser**
+
+    Create a proxy user using JWT public/private key authentication. Requires additional arguments:
+
+    * --oauth (unimplemented; reserved for future use)
+
+      Create user using OAuth authentication plugin.
+
+    * --jwt (required)
```

Comment:
If this is required, doesn't that prevent us from adding OAuth support?

Reply:
I'm not sure really what word to use that says "use this option because the command won't do anything without it, tough technically it is optional"

Reply:
If we decided to give the user a choice it would have to change from "required" to "optional" or some world that means "either this --oauth"

Reply:
Sure, see my previous comment about OAuth support.